javascript/ql/src/semmle/javascript/dataflow/DataFlow.qll

```diff
@@ -512,6 +522,22 @@ module DataFlow {
 
   /**
    * A data flow node that writes to an object property.
+   *
+   * For example, all of the following are property writes:
+   * ```js
+   * obj.f = value;  // assignment to a member
```

Comment:
Why "member" and not "property"?

Reply:
It just felt a bit cyclic, but maybe property is better to consistency

Reply:
Replaced with `property`